streams/tests/test_dask.py

```diff
@@ -0,0 +1,85 @@
+from operator import add
+
+from tornado import gen
+
+from streams.dask import scatter
+from streams import Stream
+
+from distributed import Future, Client
+from distributed.utils import sync
+from distributed.utils_test import gen_cluster, inc, cluster, loop  # flake8: noqa
+
+
+@gen_cluster(client=True)
+def test_map(c, s, a, b):
+    source = Stream()
+    futures = scatter(source).map(inc)
+    futures_L = futures.sink_to_list()
+    L = futures.gather().sink_to_list()
+
+    for i in range(5):
+        yield source.emit(i)
```

Comment:
is it possible to not have to yield? It would be nice not to have to worry about that. I am not very familiar with asynchronous environments yet (if you have recommended documentation I'd be happy to read)